### PR TITLE
[WIP] Add except if geojson_crs_def cannot be defined

### PR DESCRIPTION
**Pull request by sjib**

### CODE REVIEW

**Comment by sjib** [comment]
qgepqwat2ili/qgepdss/export.py:3800:9: E722 do not use bare 'except'

File: qgepqwat2ili/qgepdss/export.py [hunk 1]
```diff
@@ -3795,63 +3795,69 @@ def textpos_common(row, t_type, geojson_crs_def):
...
-                )
 
-            elif layer_name == "vw_qgep_wastewater_structure":
-                if obj_id not in tid_for_obj_id["abwasserbauwerk"]:
-                    logger.warning(
-                        f"Label for abwasserbauwerk `{obj_id}` exists, but that object is not part of the export"
+                if layer_name == "vw_qgep_reach":
+                    if obj_id not in tid_for_obj_id["haltung"]:
+                        logger.warning(
+                            f"Label for haltung `{obj_id}` exists, but that object is not part of the export"
```
Comment (ponceta) [inline qgepqwat2ili/qgepdss/export.py:3818]:
```suggestion
                            f"Label for reach`{obj_id}` exists, but that object is not part of the export"
```

File: qgepqwat2ili/qgepdss/export.py [hunk 2]
```diff
@@ -3795,63 +3795,69 @@ def textpos_common(row, t_type, geojson_crs_def):
...
-                    **textpos_common(label, "abwasserbauwerk_text", geojson_crs_def),
-                    abwasserbauwerkref=tid_for_obj_id["abwasserbauwerk"][obj_id],
-                )
 
-            elif layer_name == "catchment_area":
-                if obj_id not in tid_for_obj_id["einzugsgebiet"]:
+                elif layer_name == "vw_qgep_wastewater_structure":
+                    if obj_id not in tid_for_obj_id["abwasserbauwerk"]:
+                        logger.warning(
+                            f"Label for abwasserbauwerk `{obj_id}` exists, but that object is not part of the export"
```
Comment (ponceta) [inline qgepqwat2ili/qgepdss/export.py:3829]:
```suggestion
                            f"Label for wastewater structure `{obj_id}` exists, but that object is not part of the export"
```

File: qgepqwat2ili/qgepdss/export.py [hunk 3]
```diff
@@ -3795,63 +3795,69 @@ def textpos_common(row, t_type, geojson_crs_def):
...
+                        continue
+                    ili_label = ABWASSER.abwasserbauwerk_text(
+                        **textpos_common(label, "abwasserbauwerk_text", geojson_crs_def),
+                        abwasserbauwerkref=tid_for_obj_id["abwasserbauwerk"][obj_id],
+                    )
+
+                elif layer_name == "catchment_area":
+                    if obj_id not in tid_for_obj_id["einzugsgebiet"]:
+                        logger.warning(
+                            f"Label for einzugsgebiet `{obj_id}` exists, but that object is not part of the export"
```
Comment (ponceta) [inline qgepqwat2ili/qgepdss/export.py:3840]:
```suggestion
                            f"Label for catchment area`{obj_id}` exists, but that object is not part of the export"
```

**Comment by sjib** [comment]
@ponceta - I made it german (english) instead of just german or just english. OK?

**Comment by ponceta** [comment]
> @ponceta - I made it german (english) instead of just german or just english. OK?

I think more generally for logging we should stick to English and then find a way to translate error / log messages, otherwise we will have to provide every supported language and not just English + German.

But as this is more about fixing a bug here let's go that way for now and check on a more general way on TWW.

**Review by ponceta — APPROVED** [review]

**Pull request merged**